spec/requests/auth/session_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Auth::Sessions', type: :request do
+  it "creates a Session" do
+    user = create(:user)
+    post "/auth/login", :params => { 
+      :user => {
+        :email => user.email,
+        :password => user.password
+      } 
```

Comment:
In this case it would be better to user a newer syntax:
```suggestion
    post auth_login_url, params: { 
      user: {
        email: user.email,
        password: user.password
      } 
```